src/SFA.DAS.AANHub.Application/Employers/Queries/GetEmployerMemberQueryHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Employers.Queries
+{
+    public class GetEmployerMemberQueryHandler : IRequestHandler<GetEmployerMemberQuery, ValidatableResponse<GetEmployerMemberResult>>
+    {
+        private readonly IEmployersReadRepository _employersReadRepository;
+
+        public GetEmployerMemberQueryHandler(IEmployersReadRepository employersReadRepository) => _employersReadRepository = employersReadRepository;
+
+        public async Task<ValidatableResponse<GetEmployerMemberResult>> Handle(GetEmployerMemberQuery request, CancellationToken cancellationToken)
+        {
+            var employer = await _employersReadRepository.GetEmployerByUserId(request.UserId);
```

Comment:
This is more of an architecture question to follow up on, but how do we ensure we bring back the correct user?
The way this table is designed is that we can only have one pair of `accountId` and `userId`. But you could have potentially x amount of users with the userId of `1234`, for example.

So how is this managed?

Reply:
That is a good question. I think adding another parameter for accountId in this GET request should resolve that.

Reply:
The endpoint will be trickier, `/employers/{accountId}/users/{userid}` perhaps @alandburns ?